forum/csakajs/gyak.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @typedef {Object} User
+ * @property {number} id
+ * @property {string} first_name
+ * @property {string} last_name
+ * @property {string} email
+ * @property {number} birthdate
+ */
+
+////////////////////read csv
+const fs = require("fs");
+const { parse } = require("path");
+fs.readFile("csv.csv", "utf8", (err, data) => {
+  if (err) {
+    console.error(err);
+    return;
+  }
+  getArray(data, 2);
+});
+////////////////////get array
+
+function getArray(data) {
```

Comment:
```suggestion
/** Get array */
function getArray(data) {
```
But unnecessary again.